docs/swagger_ui_bravado.py

```diff
@@ -0,0 +1,47 @@
+from . import body_generation
+
+
+def generate_sample(op_name, op_spec, data_params, full_spec):
+    BASE_SAMPLE = """ 
+from bravado.requests_client import RequestsClient
```

Comment:
Can we delegate this logic to Jijnja template engine? It looks like it's supposed to do exactly what we are doing here in code.

Reply:
done